index.html

```diff
@@ -269,7 +269,7 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
           </div>
         </div>
       </section>
-      <section class="contact-card-section">
+      <form action="./index.html" class="contact-card-section" method="POST">
```

Comment:
- [ ] I can see that your form is still not submitted successfully with valid input values. and it's because you've used a wrong attribute action value it should be something else than `./index.html`
eg' `<form action="https://formspree.io/f/xnqlovnj" method="post" id="form">`

> 1. Action value should be similar to https://formspree.io/f/{form_id}